waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -44,18 +44,15 @@
     private ClientCli(String[] args,  boolean useByTest) {
         super(args, useByTest, Arrays.asList(
                 new Subcommand(Validate.NAME, Validate.DESCRIPTION, Validate::new),
-                new Subcommand(HighWaterMark.NAME, HighWaterMark.DESCRIPTION, HighWaterMark::new)
-        ));
+                new Subcommand(HighWaterMark.NAME, HighWaterMark.DESCRIPTION, HighWaterMark::new),
+                new Subcommand(Producer.NAME, Producer.DESCRIPTION, Producer::new),
+                new Subcommand(Consumer.NAME, Consumer.DESCRIPTION, Consumer::new)
```

Comment:
nit: `CreateProducer` and `CreateConsumer` instead of `Producer` and `Consumer`.